includes/general.php

```diff
@@ -2346,6 +2346,7 @@ function pods_no_conflict_on( $object_type = 'post', $object = null ) {
 				$no_conflict['filter'] = array_merge( $no_conflict['filter'], array(
 					array( 'add_post_metadata', array( PodsInit::$meta, 'add_post_meta' ), 10, 5 ),
 					array( 'update_post_metadata', array( PodsInit::$meta, 'update_post_meta' ), 10, 5 ),
+					array( 'update_post_metadata_by_id', array( PodsInit::$meta, 'update_post_meta_by_id' ), 10, 4 ),
```

Comment:
* :no_entry_sign: Short array syntax must be used to define arrays
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: a1b26ff26ed75a4ec5ec67ecac0a65a8</sub>